srcpkgs/removed-packages/template

```diff
@@ -695,6 +695,7 @@ replaces="
  vte290<=0.36.5_5
  vte<=0.28.2_16
  w_scan<=20170107_1
+ weston-colord<=9.0.0_1
```

Comment:
This should go to `weston` instead.

Reply:
should I squash the removed weston colord into the main update weston commit?